libraries/AP_Scripting/docs/docs.lua

```diff
@@ -340,6 +340,307 @@ function efi:get_state() end
 ---@return AP_EFI_Backend_ud
 function efi:get_backend(instance) end
 
+-- desc
+---@class airspeed
+airspeed = {}
+
+-- pass in external airspeed data
+---@param param1 airspeed_data_message_t_ud
+function airspeed:handle_external(param1) end
+
+
+-- desc
+---@class ExternalAHRS
+ExternalAHRS = {}
+
+-- desc
+---@param param1 EAHRS_State_ud
+---@param param2 nav_filter_status_flags_t_ud
+---@return boolean
+function ExternalAHRS:handle_scripting(param1, param2) end
+
+-- desc
+---@class mag_data_message_t_ud
+local mag_data_message_t_ud = {}
+
+---@return mag_data_message_t_ud
+function mag_data_message_t() end
```

Comment:
It might be worth a rename on this, its the creation function so you would have to rename the whole object. 

Its a little late now, but I wonder it we would have been better to namespace these userdata. Currently you do:
```
data = mag_data_message_t()
```
We could name space it so the creation is not global:
```
data = compass:mag_data_message_t()
```